### PR TITLE
Add tags filter to rest API

### PR DESCRIPTION
**Pull request by szok**
Add tags filter to rest API

### CODE REVIEW

**Comment by mkurek** [comment]
:ok:

**Comment by mkurek** [comment]
@szok btw could you add some info about searching for tags to the docs?

**Comment by ghost** [comment]
:ok:

File: docs/user/api.md [hunk 1]
```diff
@@ -126,6 +126,12 @@ results in:
 
 Part of response was skipped for readability.
 
+## You can search records by tags:
+
+`curl https://<YOUR-RALPH-URL>/api/data-center-assets/?tag=tag1&tag=tag2 | python -m json.tool`
+
+You will find all records that contain the selected tags.
```
Comment (mkurek) [inline docs/user/api.md]: `that contain the selected tags.` -> `that contain all of the specified tags.`

**Pull request merged**